lib/Payable/PayableBuilder.php

```diff
@@ -15,4 +15,20 @@ private function buildPayable($payableData)
 
         return new Payable(get_object_vars($payableData));
     }
+
+    /**
```

Comment:
@williamokano in this docblock:

- `$payablesData` is an object or an array?
- The expected `@return` is an object or an array? It should be an array, as we can see inside the method.